### ISSUE TITLE
SVG with aria-label still showing error

### ISSUE THREAD

**Issue by mmb19** [ThePacielloGroup/ARC-Tracker#6]
### Summary

When running a test on an image it says the aria-label is missing, when it's actually there

### Expected result

A passed test

### Actual result

An error; failed test

### Test Case

![image](https://user-images.githubusercontent.com/50637386/76975444-eb815d00-6908-11ea-84fa-d1d1c205cec5.png)

#### ARC Toolkit version

Version: 3.3.2.0

#### Operating System and version

Windows 10 Home

#### Browser and version:

Chrome Version 79.0.3945.130 (Official Build) (64-bit)

**Comment by sumegha26** [comment]
Hi @mmb19 ,

I was facing same issue and I was able to find out what is the issue exactly and resolved it.

Can you share your entire element screenshot ? ( not only svg ). If you are using any `div` / `button` element or element which is nesting `svg` inside  it.

**Comment by mmb19** [comment]
Sorry, I no longer use this since I'm now using axe-monitor. If this has been resolved it can be closed out.
